server/ServerFSM.cpp

```diff
@@ -2531,7 +2531,8 @@ sc::result ProcessingTurn::react(const ProcessTurn& u) {
     }
 
     if (server.IsHostless() && GetOptionsDB().Get<bool>("save.auto.hostless.enabled")) {
-        boost::filesystem::path autosave_dir_path = GetServerSaveDir() / "auto";
+        std::string subdir = GetUniverse().GameUID();
+        boost::filesystem::path autosave_dir_path = GetServerSaveDir() / (subdir.empty() ? "auto" : subdir);
```

Comment:
Does this need to be in a separate directory, rather than putting the (u)uid in the filename?

Reply:
It think it's simpler to search files organized in folders than search them by name component in the one big folder.

Reply:
Are restrictions on numbers of autosaves yet supported for hostless? Does putting them in separate subdirectories complicate that?

Reply:
No, hostless mode doesn't support restrictions on numbers of autosaves.